examples/12-upload.php

```diff
@@ -118,11 +115,12 @@
 };
 
 // buffer and parse HTTP request body before running our request handler
-$server = new Server(new MiddlewareRunner(array(
-    new RequestBodyBufferMiddleware(100000), // 100 KB max
-    new RequestBodyParserMiddleware(),
-    $handler
-)));
+$server = new Server(
+    $handler,
+    [
+        'buffer-size-limit' => 100000,
+    ]
```

Comment:
Not opposed to this, but maybe remove this whole parameter for now (see also below) and add this to a follow-up PR? This can easily be worked around by using `ini_set()` afaict.

Reply:
Ok I'll remove it so you can file your PR's faster :+1:

Reply:
Removed it :+